addons/mrp/report/report_stock_forecasted.xml

```diff
@@ -3,15 +3,15 @@
     <template id="mrp_report_product_product_replenishment" inherit_id="stock.report_product_product_replenishment">
         <xpath expr="//tr[@name='draft_picking_in']" position="after">
             <tr t-if="docs['draft_production_qty']['in']" name="draft_mo_in">
-                <td colspan="2">Draft MO</td>
+                <td colspan="2">Produce by Draft MO</td>
```

Comment:
*Produce* or *Produce*__d__ ? I'm not sure but I think it's Produced :thinking:

Reply:
Produced describes the past, which is not the case. But produce is not better. "Production of Draft MO" and "Component of Draft MO", what do you think ?

Reply:
Yep I think it's good